en/messages.json

```diff
@@ -328,6 +346,9 @@
     "popupPhoneMaskingPromoBody": {
         "message": "With phone number masking, you can now create a phone number mask that helps you get texts and calls without revealing your true number."
     },
+    "popupPhoneMaskingPromoBodyPremium": {
+        "message": "Get texts and calls without revealing your true number."
+    },
```

Comment:
<img width="164" alt="image" src="https://user-images.githubusercontent.com/13066134/190509095-cc3277ea-1cd3-4512-8de3-0828a9d7a2ba.png">
Headline for premium + non-premium is the same, but the body is shorter for the premium view.

Reply:
I was going to change the older string to `popupPhoneMaskingPromoBodyFreePlan` but I didn't want to waste the localizers' efforts translating that string since it's still correct. I'll need to devise a naming convention for these announcements in the future, since there are random discrepanies between premium/non-premium